nfafrontend/src/app/app.module.ts

```diff
@@ -1,102 +1,108 @@
-import { BrowserModule } from '@angular/platform-browser';
```

Comment:
what happened here. Why are there 1000 chars changed, where basically nothing is different?

Reply:
Have no idea, I have resolved some conflicts inthis file with git tool in eclipse, only added some new imports